specification/gedcom-2-data-types.md

```diff
@@ -30,6 +30,18 @@ Negative integers are not supported by this specification.
 
 The URI for the `Integer` data type is `xsd:nonNegativeInteger`.
 
+## Decimal
```

Comment:
Perhaps "Float" would be a better name for this proposed type

Reply:
Hi Dave, first thanks for the comments!

About Float, yes, I thought about that. But here this type is only meant to hold the lenght/distance, not the result of a very complicated calculation, so thats why I took Decimal. Should I change that into Float?